static/app/views/issueList/header.tsx

```diff
@@ -96,11 +96,7 @@ function IssueListHeader({
   function trackTabClick(tabQuery: string) {
     // Clicking on inbox tab and currently another tab is active
     if (tabQuery === Query.FOR_REVIEW && query !== Query.FOR_REVIEW) {
-      trackAnalyticsEvent({
-        eventKey: 'inbox_tab.clicked',
-        eventName: 'Clicked Inbox Tab',
-        organization_id: organization.id,
-      });
+      trackAdvancedAnalyticsEvent('inbox_tab.clicked', {organization, query: tabQuery});
```

Comment:
adding the tab query because it's helpful

Reply:
Isn't `tabQuery` always the same here? `is:unresolved is:for_review assigned_or_suggested:[me, none]`

Reply:
@malwilley well you could look at the ignored tab as well

Reply:
Hmm as far as I can tell this is only firing when the "For Review" tab is clicked. Doesn't look like we are tracking any of the others 🤔

Reply:
@malwilley you're right...I'm looking at a different event. Why do we even need this event lol?

Reply:
Probably don't anymore, was probably used when the feature initially shipped. Doubt anyone would care if it was removed hah